solidity/random-beacon/hardhat.config.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-local-networks-config"
+import "hardhat-deploy"
+import "@tenderly/hardhat-tenderly"
+
```

Comment:
Let's remember about `@typechain/hardhat` comment from [the other PR](https://github.com/keep-network/keep-core/pull/2612/files#r716568015).

Reply:
I added it in https://github.com/keep-network/keep-core/pull/2630